src/main/java/frc/utils/controllers/keyboard/KeyboardController.java

```diff
@@ -0,0 +1,151 @@
+package frc.utils.controllers.keyboard;
+
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
+import frc.utils.CMDHandler;
+import frc.utils.dashboard.LoggedTableBoolean;
+
+import java.nio.file.Path;
+
+
+/*
+ * Inspired from Trigon code.
+ *
+ * @author Yoni Kiriaty, Trigon
+ *
+ */
+public class KeyboardController {
+
+    private static final double KEY_PRESSED_VALUE = 0.5;
+
+    private static final Path KEYBOARD_TO_NETWORK_TABLES_CLASS = Path.of("KeyboardToNetworkTables");
+
+    private static final String KEYBOARD_TABLE = "Keyboard";
+    private static final String KEYS_TAB = "Keys/";
+
+    public final Trigger
+            ESC,
+            F1, F2, F3, F4, F5, F6, F7, F8, F9, F10, F11, F12,
+            DELETE, BACKTICK,
+            ONE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE, ZERO,
+            MINUS, EQUALS, BACKSPACE, TAB,
+            A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P, Q, R, S, T, U, V, W, X, Y, Z,
+            SEMICOLON, APOSTROPHE, LEFT_SHIFT, COMMA, PERIOD, SLASH, RIGHT_SHIFT, LEFT_CONTROL, LEFT_ALT, RIGHT_CONTROL,
+            LEFT_ARROW, RIGHT_ARROW, UP_ARROW, DOWN_ARROW,
+            NUMPAD_0, NUMPAD_1, NUMPAD_2, NUMPAD_3, NUMPAD_4, NUMPAD_5, NUMPAD_6, NUMPAD_7, NUMPAD_8, NUMPAD_9;
+
+    public KeyboardController() {
+        if (Robot.isSimulation()) {
+            CMDHandler.runPythonClass(KEYBOARD_TO_NETWORK_TABLES_CLASS, "127.0.0.1");
```

Comment:
```suggestion
             Computer/SimulationIP = "127.0.0.1";
            CMDHandler.runPythonClass(KEYBOARD_TO_NETWORK_TABLES_CLASS, Computer/SimulationIP);
```
and put it in const file or something